mentors/delphi/0.json

```
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "Neftali",
+    "name": "Germán Estévez -Neftalí-",
+    "link_text": "Personal website about Delphi",
+    "link_url": "https://neftali.clubdelphi.com/",
+    "avatar_url": "https://neftali.clubdelphi.com/wp-content/uploads/Foto_German_peque2.jpg",
+    "bio": "I like the programming, and although I have programmed in different languages, until now the one that has captivated me has been Delphi. I am what is known as a "specialist". I have been using Delphi (since version 3) for more than 20 years."
```

```suggestion
    "bio": "I like the programming, and although I have programmed in different languages, until now the one that has captivated me has been Delphi. I am what is known as a 'specialist'. I have been using Delphi (since version 3) for more than 20 years."
```
Double quotes are confusing things.